docs/development/getting_started.rst

```diff
@@ -145,7 +145,7 @@ development, staging, and prod (production).
    hardening applied but virtualized
 
 If you plan to alter the configuration of any of these machines, make sure to
-review the `Development Guide for Serverspec Tests </docs/spec_tests.md>`__.
+review the `Development Guide for Serverspec Tests </docs/development/spec_tests.rst>`__.
```

Comment:
Nope, this doesn't work. Did you even try building this locally to make sure that it worked?

First of all, when Sphinx renders this documentation, it creates **HTML** (`.html`) from ReStructuredText (`.rst`). So this link ending in `.rst` is bogus, and cannot be reasonably expected to return anything but a 404.

Historical background: before the switch to RTD, we relied on GitHub's automatic rendering of Markdown files to have presentable documentation for end users. As a result, we started using relative and root-relative links to achieve inter-documentation hyperlinking. This was a gross hack that was totally dependent on GitHub's (fortunately stable and consistent) URL scheme.

Now that we're using Sphinx (the builder underpinning RTD), we can *explicitly* refer to other parts of the documentation, instead of resorting to gross (*implicit*) hacks. I have already started doing this in some of the documentation that I migrated. You should use the [`:doc:`](http://sphinx-doc.org/markup/inline.html#role-doc) role to refer to another document, or the [`:ref:`](http://sphinx-doc.org/markup/inline.html#role-ref) role to refer to a section of another document.